frontend/src/app/account-activation/account-activation.component.ts

```diff
@@ -0,0 +1,40 @@
+import { Component, OnInit } from '@angular/core';
+import { AccountActivationRequestModel } from 'app/auth.models';
+import { Observable } from 'rxjs/Observable';
+import { AuthService } from '../auth.service';
+import { Router, ActivatedRoute, Params } from '@angular/router';
+
+@Component({
+  selector: 'volontulo-account-activation',
+  templateUrl: './account-activation.component.html',
+  styleUrls: ['./account-activation.component.css']
```

Comment:
Two things here:
1. we start using `scss`, not raw `css` for styles
1. we agreed not to store empty styleshits in repository